examples/js/exporters/GLTFExporter.js

```diff
@@ -1165,6 +1165,30 @@ THREE.GLTFExporter.prototype = {
 				var attribute = geometry.attributes[ attributeName ];
 				attributeName = nameConversion[ attributeName ] || attributeName.toUpperCase();
 
+				// Prefix all geometry attributes except the ones specifically
+				// listed in the spec; non-spec attributes are considered custom.
+				var validVertexAttributes = [
+					/^POSITION$/,
+					/^NORMAL$/,
+					/^TANGENT$/,
+					/^TEXCOORD_\d+$/,
+					/^COLOR_\d+$/,
+					/^JOINTS_\d+$/,
+					/^WEIGHTS_\d+$/,
+				];
```

Comment:
A single long regexp would let us shorten this list, and for/if blocks below, into a single if statement. For conciseness I think that's worth it.

```
/^(POSITION|NORMAL|TANGENT|TEXCOORD_\d+|COLOR_\d+|JOINTS_\d+|WEIGHTS_\d+)$/
```